terraform/testcases/standard_otlp_metric_trace/otconfig.tpl

```diff
@@ -0,0 +1,31 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: debug
```

Comment:
There is a new flag in the test framework that will control this. see other configs for an example.

Reply:
Can you point me to this new flag in the test framework?  I can't seem to find it in the other configs

Reply:
https://github.com/aws-observability/aws-otel-test-framework/blob/23f605c6125b4465c5961b292f01f8bccf228e7d/terraform/testcases/otlp_trace/otconfig.tpl#L29

Reply:
Although that is being used to control the telemetry log level and not the logging exporter. I am not sure if we should be using it for both. So I may be off base here.